lib/success.js

```diff
@@ -59,7 +60,7 @@ module.exports = async (pluginConfig, context) => {
 			// truncate long messages
 			let messageText =
 				nextRelease.notes.length > MAX_LENGTH
-					? nextRelease.notes.substring(0, MAX_LENGTH) + '*[...]*'
+					? truncate(nextRelease.notes, MAX_LENGTH)
```

Comment:
Is this neccesary? As `truncate()` alreade checks the length this could be just one row.

Reply:
Nope, you're right!